modules/issue_tracker/php/edit.class.inc

```diff
@@ -424,6 +414,29 @@ class Edit extends \NDB_Page implements ETagCalculator
         return $whoIsWatching;
     }
 
+    /**
+     * Gets the description of the issue
+     *
+     * @param int $issueID the relevant issue
+     *
+     * @return string
+     */
+    function getDescription(int $issueID): string
+    {
+        $db = $this->loris->getDatabaseConnection();
+
+        $desc = $db->pselect(
+            "SELECT description from issues WHERE issueID=:issueID",
+            ['issueID' => $issueID]
+        );
+
+        $description = "";
+        foreach ($desc as $d) {
+            $description = $d['description'] ?? '';
+        }
+        return $description;
+    }
```

Comment:
I'm a bit confused about what's going on here, why is the description being re-set every time in a foreach loop? Wondering if something can be done to the database query to select the description instead?